utils/dataset_manifest/core.py

```diff
@@ -194,14 +194,18 @@ def __iter__(self):
             if idx in self.range_:
                 image = next(sources)
                 img = Image.open(image, mode='r')
+                orientation = img.getexif().get(274, 1)
```

Comment:
@Marishka17 , probably you want to look at these changes and test them.

Reply:
@nmanovic, Everything works fine.